README.md

```diff
@@ -31,6 +31,12 @@ Yarn:
 yarn add sharetribe-flex-integration-sdk
 ```
 
+## Getting client ID and client secret
+
+Client ID and secret are required to create new SDK instance. 
+See [the doc](https://www.sharetribe.com/docs/integrations/getting-started-with-integration-api/#create-integration-api-application-in-flex-console)
```

Comment:
I'd perhaps just move this under the Usage section and below the code example. So no separate heading.

Reply:
Changed as suggested